src/main/java/bogdanSandu/driver/BrowserManager.java

```diff
@@ -32,7 +35,15 @@ public static ChromeDriver getChromedriverForMobile(){
         return new ChromeDriver(chromeOptions);
     }
 
-    public static void closeBrowser(ChromeDriver driver){
+    public static RemoteWebDriver creatRemoteWebDriver(){
```

Comment:
Please also rename method from creatRemoteWebDriver() to createRemoteWebDriver(). Thanks!